ndarray-rand/src/lib.rs

```diff
@@ -35,7 +35,7 @@ use crate::rand::seq::index;
 use crate::rand::{thread_rng, Rng, SeedableRng};
 
 use ndarray::{Array, Axis, RemoveAxis, ShapeBuilder};
-use ndarray::{ArrayBase, DataOwned, RawData, Data, Dimension};
+use ndarray::{ArrayBase, Data, DataOwned, Dimension, RawData};
```

Comment:
Formatting changes for the whole crate should be in a different PR, see #922

Reply:
Yeah I tried not to commit anything that I hadn't changed but I'll go through and revert what I can. It's just a bit faffy when cargo fmt is basically part of my workflow :sweat_smile: